pages/templates/pages/home.html

```diff
@@ -9,7 +9,7 @@ <h2>Home</h2>
 
 <div class="row pb-5">
     <div class="col-12">
-        <div id="spg-home-carousel" class="carousel slide" data-ride="carousel">
+        <div id="spg-home-carousel" class="carousel slide" data-ride="carousel" data-interval="5000">
```

Comment:
Isn't 5000 the default?  If so, then it's strange why this would fix the issue.
https://getbootstrap.com/docs/4.0/components/carousel/

Reply:
I thought the same thing.  But it behaves differently

Reply:
I thought the same thing.  It's weird

Reply:
hmm.  I actually played around with it a bit more and it's not so simple.  I wish we could get an indicator telling us when it resets.  Sometimes when I scroll it resets and sometimes it doesn't.  Maybe it has to do with whether it's on the screen or not?  Regardless, we probably don't have to worry about that in this merge